.github/workflows/integration.yml

```diff
@@ -22,15 +22,13 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
       - name: Install Build Requirements
-        run: |
-          dnf -y install git autoconf autoconf-archive automake libtool \
-            openssl-devel
+        run: |\
+          dnf -y install git meson openssl-devel
       - name: Setup, Build and Install pkcs11-provider
         run: |
-          autoreconf -fiv
-          ./configure --libdir=/usr/lib64
-          make
-          make install
+          meson setup -Dlibdir=/usr/lib64 builddir
+          meson compile -C builddir
+          meson install -C builddir
```

Comment:
Is there a way to set some variable so that -C builddir does not have to be repeated on each command ?

Reply:
Not that I know of, I'm afraid.